trace/kafka/kafka.go

```diff
@@ -11,12 +11,32 @@ import (
 	"github.com/beatlabs/patron/trace"
 	"github.com/opentracing/opentracing-go"
 	"github.com/opentracing/opentracing-go/ext"
+	"github.com/prometheus/client_golang/prometheus"
 )
 
 const (
 	producerComponent = "kafka-async-producer"
 )
 
+var messageStatus *prometheus.CounterVec
+
+// MessageStatusCountInc increments the messageStatus counter for a certain status.
+func MessageStatusCountInc(status, topic string) {
```

Comment:
Need to be exported?

Reply:
I should be asking this question myself more often. Sorry, fixing this.

Reply:
just start always with unexported...